samples/SampleStartups/StartupFullControl.cs

```diff
@@ -13,7 +14,14 @@ public class StartupFullControl
     {
         public static void Main(string[] args)
         {
+            var config = new ConfigurationBuilder()
+                .AddCommandLine(args)
+                .AddEnvironmentVariables(prefix: WebHostDefaults.EnvironmentVariablesPrefix)
+                .AddJsonFile(WebHostDefaults.HostingJsonFile, optional: true)
```

Comment:
Remove the WebHostDefaults.HostingJsonFile constant, it's no longer used in src.

I'm tempted to remove WebHostDefaults.EnvironmentVariablesPrefix as well, as it's only used once in src.